### PR TITLE
Ps 5514 Percona Server can't be installed after sysbench has been installed

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: build-ps/percona-server-8.0_builder.sh [hunk 1]
```diff
@@ -91,6 +93,13 @@ check_workdir(){
 add_percona_yum_repo(){
     if [ ! -f /etc/yum.repos.d/percona-dev.repo ]
     then
+	if [ $RHEL = 8 ]; then
```
Comment (hors) [inline build-ps/percona-server-8.0_builder.sh]: "x$RHEL" = 'x8'

File: build-ps/percona-server-8.0_builder.sh [hunk 2]
```diff
@@ -747,9 +757,13 @@ build_tarball(){
     rm -fr ${TARFILE%.tar.gz}
     tar xzf ${TARFILE}
     cd ${TARFILE%.tar.gz}
-    CMAKE_OPTS="-DWITH_ROCKSDB=1" bash -xe ./build-ps/build-binary.sh --with-mecab="${MECAB_INSTALL_DIR}/usr" --with-jemalloc=../jemalloc/ ../TARGET
-    DIRNAME="tarball"
-
+    if [ $WITH_SSL = 1 ]; then
```
Comment (hors) [inline build-ps/percona-server-8.0_builder.sh]: "x$WITH_SSL" = 'x1'

**Review by hors — APPROVED** [review]

**Comment by peterschwaller** [comment]
Dummy comment to trigger sync to Jira.  Please ignore.

**Comment by peterschwaller** [comment]
Dummy comment to trigger sync to Jira.  Please ignore.

**Pull request merged**